ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,115 @@
-﻿namespace ConsoleApp1
+﻿using System;
+using System.Linq;
+
+namespace ConsoleApp1
 {
 	internal class Program
 	{
 		static void Main(string[] args)
 		{
+			string a = "hello";
+			string b = "hello,*%";
+			string forSorting = "DCBaz";
+
+			Compare(a, b);//#1
+			Analyze(b);//#2
+			Sort(forSorting);//#3
+			Duplicate("mHellooomm");//#4 
+        }
+
+		//#1
+        //Compare that will return true if 2 strings are equal,
+		//otherwise false, but do not use build-in method
+
+        public static bool Compare(string a, string b)
+		{
+			if (a.Length != b.Length)
+			{
+				Console.WriteLine($"False, a.length = {a.Length!}, b.Length = {b.Length!}");
+				return false;
+			}
+
+			if( a != b)
+			{
+				Console.WriteLine($"{a} not equal to {b}");
+				return false;
+			}
+
+			Console.WriteLine($"True, {a} equal to {b}");
+			return true;
+		}
+
+		//#2
+		//Analyze that will return number of alphabetic chars in string,
+		//digits and another special characters
+
+		public static int Analyze(string s)
+		{
+			Console.WriteLine($"String is '{s}', length is {s.Length}");
```

Comment:
here you need to display how many digits, letters and special characters in the passed string
For instance, "12sdf***23%g#"
OutPut
Digits: 4
Letters: 4
Special characters: 5